modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/appearance.dm

```diff
@@ -0,0 +1,75 @@
+/datum/sprite_accessory/proc/lewd_is_hidden(mob/living/carbon/human/target_human, obj/item/bodypart/bodypart, hide_if_catsuit = TRUE, hide_if_sleeping_bag = TRUE)
+	if(target_human.wear_suit && istype(target_human.wear_suit, /obj/item/clothing/suit/straight_jacket/kinky_sleepbag))
```

Comment:
Literally why are we doing this this way, rather than just using the existing checks to see if they should be obscured?

Reply:
I have no idea, honestly.

Reply:
After having a closer look, this code makes more sense. It isn't ideal, but it's to ensure catsuits, sleeping bags, and their subtypes hide body parts appropriately. Existing code doesn't and can't cover this.